### ISSUE TITLE
[BUG] cellfinder_view throws a brainrender error "No module named 'brainrender.Utils.image"

### ISSUE THREAD

**Issue by rdbcasillas** [SainsburyWellcomeCentre/cellfinder#134]
**Describe the bug**
When I run cellfinder_view , I get the following error: 

```
(cellfinder) C:\Users\Administrator>cellfinder_view
Traceback (most recent call last):
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Administrator\miniconda3\envs\cellfinder\Scripts\cellfinder_view.exe\__main__.py", line 4, in <module>
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\site-packages\neuro\visualise\viewer.py", line 23, in <module>
    from neuro.visualise.napari_tools.callbacks import display_brain_region_name
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\site-packages\neuro\visualise\napari_tools\callbacks.py", line 7, in <module>
    from neuro.segmentation.manual_segmentation.man_seg_tools import (
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\site-packages\neuro\segmentation\manual_segmentation\man_seg_tools.py", line 15, in <module>
    from neuro.visualise.brainrender_tools import (
  File "c:\users\administrator\miniconda3\envs\cellfinder\lib\site-packages\neuro\visualise\brainrender_tools.py", line 3, in <module>
    from brainrender.Utils.image import reorient_image, marching_cubes_to_obj
ModuleNotFoundError: No module named 'brainrender.Utils.image'
```

**Desktop (please complete the following information):**
Windows 10

**Comment by adamltyson** [comment]
Hi, sorry about this, what version of brainrender do you have? If it's `1.0.0.0` then this functionality might have been removed, @FedeClaudi?

I'll look into fixing this, but if you do have `1.0.0.0` it might be fixed temporarily by installing an older version of brainrender (`pip install brainrender==0.4.0.0`).

Let me know how you get on.

**Comment by rdbcasillas** [comment]
Thanks @adamltyson, going back to 0.4.0.0 worked!

**Comment by FedeClaudi** [comment]
Hi, 

That's right I removed that function, sorry. 
I thought you had implemented similar functionality in the neuro package @adamltyson ? If you need it back let me know and I'll add it back in, sorry I should've double checked with you before removing it.

**Comment by adamltyson** [comment]
No problem @FedeClaudi. Reopening to remind me to add this functionality into neuro.

**Comment by FedeClaudi** [comment]
okay, thanks and sorry again :)

**Comment by adamltyson** [comment]
Hi @rdbcasillas, this should be fixed by installing `pip install neuro==0.0.19`
